aci/resource_aci_cloudextepg.go

```diff
@@ -168,7 +168,7 @@ func getRemoteCloudExternalEPg(client *client.Client, dn string) (*models.CloudE
 	cloudExtEPg := models.CloudExternalEPgFromContainer(cloudExtEPgCont)
 
 	if cloudExtEPg.DistinguishedName == "" {
-		return nil, fmt.Errorf("CloudExternalEPg %s not found", cloudExtEPg.DistinguishedName)
+		return nil, fmt.Errorf("Cloud External EPg %s not found", dn)
```

Comment:
EPg -> EPG

Reply:
See this is marked as resolved but is still EPg? See other usage of EPG so should this not be changed?